uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -50,10 +63,14 @@ <h4 class="modal-title" ng-switch="documentModal.isRole('admin')">
       <div class="tab-content">
         <div class="tab-pane active" id="detailsTab">
           <!-- Drop Zone -->
-          <br>
+          <br ng-if="!documentModal.isRole('staff')">
           <div class="row">
             <div class="col-xs-12">
-              <div class="well drop-zone" ng-click="documentModal.dropzoneClick()">
+              <label>Document File</label>
+              <div class="well drop-zone"
+                ng-if="documentModal.isRole('admin') || (documentModal.isRole('staff') && documentModal.mode == 'edit')"
+                ng-class="{'no-file': !documentModal.uploader.queue.length && !documentModal.files.length && documentModal.isRole('staff') && !documentModal.containsFiles && documentModal.mode == 'edit'}"
```

Comment:
cant we use `containsFiles` instead of `!documentModal.files.length && documentModal.isRole('staff') && !documentModal.containsFiles`